{{cookiecutter.module_name|slugify}}/setup.cfg

```
@@ -14,6 +14,9 @@ packages=find_namespace:
 install_requires =
     inmanta-module-std
 
+[options.packages.find]
+include = inmanta_plugins
```

Otherwise submodules will not get packaged
```suggestion
include = inmanta_plugins*
```